### ISSUE TITLE
issuse:"save the tool not work"

### ISSUE THREAD

**Issue by shenyunhuan** [EtiennePerot/open-webui-code-execution#20]
- **Open WebUI version**: 0.3.28
- **Tool/function version**: 0.6.0
issuse info "Something went wrong :/ 'gbk' codec can't decode byte 0xa6 in position 57656: illegal multibyte sequence"

**Comment by EtiennePerot** [comment]
It sounds like you are using a non-UTF-8 encoding inside the Open WebUI container.

Can you run the following commands inside the container? (Or, if you are not running Open WebUI in a container, run them wherever Open WebUI runs):

```
$ env | grep -E 'LANG|LC'
$ cat /etc/locale.conf
$ cat /etc/locale.gen
$ locale
$ python3 -c 'import sys; print(sys.stdout.encoding)'
```

**Comment by EtiennePerot** [comment]
I believe this is fixed in v0.7.0.
